basyx.submodelrepository/basyx.submodelrepository-feature-operation-delegation/Readme.md

```diff
@@ -0,0 +1,34 @@
+# Submodel Repository - Operation Delegation
+An operation is a specific type of submodel element capable of performing server-side functions. However, in certain environments, this is not possible. Therefore, the operation can be delegated to an independent destination (such as an HTTP URL) to be executed. An example request body is given below.
+```
+{
+	"inputArguments": [
+		{
+		    "value": {
+		        "modelType": "Property",
+		        "valueType": "xs:int",
+		        "value": "5",
+		        "idShort": "input"
+		    }
+	 	}
+	]
+}
+```
+
+This feature is enabled by default and to disable this the following property should be defined:
+
+```
+basyx.submodelrepository.feature.operation.delegation.enabled = false
+```
+
+To create an operation delegation, a Submodel that has an Operation(-SubmodelElement) must be present:
+-	The Operation has neither input, output nor inout variables
```

Comment:
Why shouldn't the operation have input/output variables?

Reply:
This was unnecessary, hence removed it.